iroha/src/lib.rs

```diff
@@ -95,8 +96,8 @@ pub struct Iroha {
 impl Iroha {
     /// Default `Iroha` constructor used to build it based on the provided `Configuration`.
     pub fn new(config: &Configuration, permissions_validator: PermissionsValidatorBox) -> Self {
-        iroha_logger::init(&config.logger_configuration).expect("Failed to initialize logger.");
-        log::info!("Configuration: {:?}", config);
+        iroha_logger::init(config.logger_configuration);
+        iroha_logger::info!(?config, "Started");
```

Comment:
Is it a correct syntax?

Reply:
Yeah, it is. More info could be found here:
https://docs.rs/tracing/0.1.25/tracing/#recording-fields